sdk/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3 // indirect
 	golang.org/x/sys v0.0.0-20191008105621-543471e840be
 	golang.org/x/text v0.3.1-0.20181227161524-e6919f6577db // indirect
+	google.golang.org/appengine v1.4.0 // indirect
```

Comment:
Curious: where'd this come from?

Reply:
It and `kr` like to pop up mysteriously, and have been doing so for a while. I don't think anyone has figured out the source.

Reply:
My `go mod tidy` did remove it though.

Reply:
That's weird, `go mod tidy` doesn't like to remove it for me. Must be something related to our versions of Go.
```
$ go version
go version go1.13.5 linux/amd64
```
I'll fiddle a little and see if I can get Go to minimize dependencies more.

Reply:
Oddly, after some fiddling with the other stuff, my `sdk/go.mod` file popped up as having changes from Go. It had removed that dependency. So I pushed up the removal. But why/how? :-) <shrug>